java/ql/lib/change-notes/2024-04-26-outdated-deprecations.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
```

Comment:
Shouldn't this be `breaking`?

Reply:
Probably this has been discussed in the past, sorry if that's the case.

Reply:
It is technically a breaking change to delete code. However, it's code that has been deprecated for more than a year, so external users have had plenty of to migrate. 

So I still think it's a minor change.  

I couldn't find a discussion in the past about this.  
But all previous change-notes when deleting deprecations have been `minorAnalysis`. (Since [the first deprecation deletion PR](https://github.com/github/codeql/pull/8347/)).

Reply:
I don't think the amount of time the thing has been deprecated should influence whether or not this is a breaking change. A breaking change is a breaking change.

Obviously, this breaking change is totally justified, but I still think stuff like this should be marked as a breaking change going forward